[2024-12-26T09:03:18.503+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-26T09:03:18.552+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stream_vehicle_alerts_processing.extract_data_from_kafka manual__2024-12-26T08:57:58.593307+00:00 [queued]>
[2024-12-26T09:03:18.592+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stream_vehicle_alerts_processing.extract_data_from_kafka manual__2024-12-26T08:57:58.593307+00:00 [queued]>
[2024-12-26T09:03:18.595+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2024-12-26T09:03:18.623+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): extract_data_from_kafka> on 2024-12-26 08:57:58.593307+00:00
[2024-12-26T09:03:18.630+0000] {standard_task_runner.py:72} INFO - Started process 1290 to run task
[2024-12-26T09:03:18.637+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'stream_vehicle_alerts_processing', 'extract_data_from_kafka', 'manual__2024-12-26T08:57:58.593307+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/streamVehiculeAlertsProcessing.py', '--cfg-path', '/tmp/tmpcqq1os7a']
[2024-12-26T09:03:18.645+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask extract_data_from_kafka
[2024-12-26T09:03:18.747+0000] {task_command.py:467} INFO - Running <TaskInstance: stream_vehicle_alerts_processing.extract_data_from_kafka manual__2024-12-26T08:57:58.593307+00:00 [running]> on host 3096e710f899
[2024-12-26T09:03:18.930+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stream_vehicle_alerts_processing' AIRFLOW_CTX_TASK_ID='extract_data_from_kafka' AIRFLOW_CTX_EXECUTION_DATE='2024-12-26T08:57:58.593307+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-26T08:57:58.593307+00:00'
[2024-12-26T09:03:18.933+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-26T09:03:54.250+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to success. Terminating instance.
[2024-12-26T09:03:54.256+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-26T09:03:54.263+0000] {process_utils.py:132} INFO - Sending 15 to group 1290. PIDs of all processes in the group: [1292, 1290]
[2024-12-26T09:03:54.268+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 1290
[2024-12-26T09:03:54.274+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-26T09:03:54.346+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-26T09:03:54.348+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1292, status='terminated', started='09:03:18') (1292) terminated with exit code None
[2024-12-26T09:03:54.385+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1290, status='terminated', exitcode=0, started='09:03:17') (1290) terminated with exit code 0
